### PR TITLE
SHOT-3494 - Added TEMP variable for plugins.lst file

### PR DESCRIPTION
**Pull request by rob-aitchison**
Also added fix for SHOT-3549 to add quotes around the plugins.lst file.

### CODE REVIEW

File: startup.py [hunk 1]
```diff
@@ -106,7 +106,8 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
         # Flag -P (plugins list file)
         plugins_list_file = self._get_plugins_list_file(exec_path, tk_alias_codename)
         if plugins_list_file:
-            args += " -P {0}".format(plugins_list_file)
+            args += ' -P "{0}'.format(plugins_list_file)
```
Comment (staceyoue) [inline startup.py:109]: Are you missing the closing double quote, or should it not be there in this part of the formatted string:
`' -P "{0}'`

Comment (rob-aitchison) [reply startup.py:109]: Hey @staceyoue it is on the next line as cat'ing it after that format call was not working out.

Comment (staceyoue) [reply startup.py:109]: Ah, got it!

**Review by barbara-darkshot — APPROVED** [review]
LGTM!

**Review by staceyoue — APPROVED** [review]
LGTM!

**Pull request merged**